addons/adyen_platforms/models/adyen_account.py

```diff
@@ -154,40 +172,94 @@ def _compute_full_name(self):
+                doc_type = doc_types.get(kyc.verification_type, _('Other'))
+                checks.setdefault(doc_type, []).append({
+                    'document': kyc.document,
+                    'message': kyc.status_message,
+                })
+
+            account.kyc_status_message = self.env['ir.qweb']._render('adyen_platforms.kyc_status_message', {
+                'checks': checks
+            })
+
     @api.model
     def create(self, values):
-        adyen_account_id = super(AdyenAccount, self).create(values)
-        self.env.company.adyen_account_id = adyen_account_id.id
+        self.check_access_rights('create')
 
         # Create account on odoo.com, proxy and Adyen
-        response = adyen_account_id._adyen_rpc('create_account_holder', adyen_account_id._format_data())
+        create_data = self._format_data(values)
+        create_data['payoutSchedule'] = ADYEN_PAYOUT_FREQUENCIES.get(values.get('payout_schedule', 'biweekly'), 'BIWEEKLY_ON_1ST_AND_15TH_AT_MIDNIGHT'),
+        response = self._adyen_rpc('v1/create_account_holder', create_data)
 
-        # Save adyen_uuid and proxy_token, that have been generated by odoo.com and the proxy
-        adyen_account_id.with_context(update_from_adyen=True).write({
-            'adyen_uuid': response['adyen_uuid'],
-            'proxy_token': response['proxy_token'],
-        })
+        values['account_code'] = response['adyen_response']['accountCode']
+        values['adyen_uuid'] = response['adyen_uuid']
+        values['proxy_token'] = response['proxy_token']
+
+        adyen_account_id = super(AdyenAccount, self).create(values)
+        self.env.company.adyen_account_id = adyen_account_id.id
 
-        # A default payout is created for all adyen accounts
-        adyen_account_id.env['adyen.payout'].with_context(update_from_adyen=True).create({
-            'code': response['adyen_response']['accountCode'],
-            'adyen_account_id': adyen_account_id.id,
-        })
         return adyen_account_id
 
     def write(self, vals):
-        res = super(AdyenAccount, self).write(vals)
-        if not self.env.context.get('update_from_adyen'):
-            self._adyen_rpc('update_account_holder', self._format_data())
-        return res
+        self.check_access_rights('write')
+
+        adyen_fields = {
+            'country_id', 'state_id', 'city', 'zip', 'street', 'house_number_or_name', 'email', 'phone_number',
+            'is_business', 'legal_business_name', 'doing_business_as', 'registration_number', 'first_name',
+            'last_name', 'date_of_birth', 'document_number', 'document_type',
+        }
+        if set(vals.keys()) & adyen_fields and not self.env.context.get('update_from_adyen'):
+            self._adyen_rpc('v1/update_account_holder', self._format_data(vals))
+
+        if 'payout_schedule' in vals:
+            self._update_payout_schedule(vals['payout_schedule'])
+
+        return super(AdyenAccount, self).write(vals)
```

Comment:
```suggestion
        return super().write(vals)
```
Explicit reference is not necessary unless you need to customise the `self` used for the super call.